src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -6,192 +6,186 @@ import { Callout } from "/src/components/callout"
 The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
 </Callout>
 
-By running a **verifier** (also called a **worker**) for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network for a given chain.
+By running a **verifier** (also called a **worker**) for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network and signing command batches for a given chain.
 
 ## Prerequisites
 
-1. Pull the **`tofnd`** and `ampd` image from [Docker Hub](https://hub.docker.com/):
-    ```bash
-    docker pull haiyizxx/tofnd:latest
-    docker pull haiyizxx/ampd:v0.0.2
-    ```
-1. Create an `ampd` folder in your working directory, download [`config.toml`](https://file.notion.so/f/f/8f1eb2f6-501d-4ae8-bd56-d35b21d3750f/a9bc03f8-a772-41bb-ad17-4f1b650f18ca/config.toml?id=6a8ec351-527a-467e-ab81-abc7ee8a9950&table=block&spaceId=8f1eb2f6-501d-4ae8-bd56-d35b21d3750f&expirationTimestamp=1705082400000&signature=qeG1WvtWM640SjvpYwg3ZMcBgEpCbIq2gUO1diGYbVQ&downloadName=config.toml), and put it in `ampd` folder.
-
-## Onboard as a verifier
-
-### Run `tofnd`
+1. Run `tofnd`. [`ampd`](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd#worker-onboarding), the off-chain daemon responsible for voting and signing with the amplifier protocol, needs access to a running `tofnd` instance in order to onboard as a worker or run the daemon. See the [tofnd repository](https://github.com/axelarnetwork/tofnd) for more info.
+
+2. Download `ampd` through either the [binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.1.0) or [docker](https://hub.docker.com/r/axelarnet/axelar-ampd).
+
+3. Create a `config.toml` file and put it in your `~/.ampd/` directory. 
+
+By default, `ampd` loads the `config` file from `~/.ampd/config.toml` when running any command. This can be overridden by passing `--config [path]`.
+
+The file should have the following format:
+
+```toml
```

Comment:
I would just link to the readme. The readme format could change over time, and it would be easier to just have a link here, rather than needing to constantly keep this doc up to date.